spyder/plugins/ipythonconsole.py

```diff
@@ -418,7 +419,14 @@ def setup_page(self):
                           _("Height:")+"  ", " "+_("inches"),
                           'pylab/inline/height', min_=1, max_=20, step=1,
                           tip=_("Default is 4"))
-        
+        bbox_inches_box = newcb(
+                _("Set bbox_inches to 'tight' when plotting inline"),

```

Comment:
I think this can be worded better. Something like

> Use a tight layout for inline plots

@CAM-Gerlach, what do you think?

Reply:
Yes, I like that.